File: code/R/R_Computational_Toolbox_Files/Ants_R/initPherGrid.R
initPherGrid <- function(n){
	
# function to initialize the pherGrid 

utils::globalVariables(c("MAXPHER"))

Grid = matrix(rep(0,(n+2)*(n+2)), n+2)
mid = ceiling((n+2)/2)
for(i in 1:n){

	Grid[mid,i+1]<-i/n*MAXPHER
}

return(Grid)
}

File: products/Introduction_to_Computational_Science_Modules/02_System_Dynamics/MATLAB/Files/Ants/initPherGrid.m
function grid = initPherGrid(n)
% INITPHERGRID - initialization of n+2-by-n+2 array for pheromone distribution
global MAXPHER

grid = zeros(n+2, n+2);
mid = round(n/2) + 1;
for i = 1:n
    grid(mid, i+1) = MAXPHER * i / n;
end